SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -19,11 +19,13 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, ActorReplicationRateLimit(0)
 	, EntityCreationRateLimit(0)
 	, OpsUpdateRate(1000.0f)
+	, bEnableHandover(true)
 	, bUsingQBI(true)
 	, PositionUpdateFrequency(1.0f)
 	, PositionDistanceThreshold(100.0f) // 1m (100cm)
 	, bEnableMetrics(true)
 	, MetricsReportRate(2.0f)
+	, bUseFrameTimeAsLoad(false)
```

Comment:
Looks like it's not supposed to be in this PR? I can't find any place it's used.

Reply:
This is a driveby - it wasn't defaulted previously.